### PR TITLE
Feature/responsive

### PR DESCRIPTION
**Pull request by bagnall-fetch**

### CODE REVIEW

**Review by devjsc — CHANGES_REQUESTED** [review]
Some changes required: 

Debug has been committed
Scroll bar on left hand menu is displayed in desktop mode
When screen is in mobile, text leaves the "Tutorials" link on the main page
Decreasing gap between menu items on desktop would be preferable. 

Please also merge master in before commit too

**Comment by bagnall-fetch** [comment]
The following work was carried out in this PR:

Rework header component to accept child menu items.
Rework ambassador form to work with mailchimp (rename inputs to match, change radio to checkbox)
Create CheckBox component.
Add href to footer
Modify footer styling to sync with other sites.
Modify header styling to sync with other sites.
Change to hero text.
Fix car animation not to crash on smaller devices.
Fix id/name in TextField and TextArea.
Fix grid layout in TextField and TextArea.
Change video to be IOT video.
Rework styling breakpoints to be mobile first.
Adjust M 't-shirt size' breakpoint.

**Comment by bagnall-fetch** [comment]
> Some changes required:
> 
> Debug has been committed

debug removed

> Scroll bar on left hand menu is displayed in desktop mode

reworked to remove (not a simple fix unfortunately)

> When screen is in mobile, text leaves the "Tutorials" link on the main page

Which size mobile please?

> Decreasing gap between menu items on desktop would be preferable.

I think we should create a ticket for this as it will involve all websites for consistency. And are we talking no-gap at all?

> 
> Please also merge master in before commit too

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (cyenyxe) [inline .vscode/settings.json]: Please delete and add to `.gitignore`.

**Review by devjsc — CHANGES_REQUESTED** [review]
Between 960 and 881px the header bar has a larger height than any other menu and needs to be set with a smaller height. 

The Text "Type to search" in the search bar is aligned with content in full screen but loses it's alignment when the menu bar is hidden.

File: docs/index.md [hunk 2]
```diff
@@ -1,21 +1,22 @@
...
-      <h3>Quickstart</h3>
-      <p> If you want to get moving quickly, this is the place.  </p>
-    </div>
-  </a>
-  
-  <a href="./tutorials/" class="grid-item item3">
-      <h3>Tutorials</h3>
-            
-      <p> Once you're all setup with Fetch.ai, go here to see and build following excellent tutorials. </p>
+    <a id="qs" href="getting-started/quickstart/">
```
Comment (cyenyxe) [inline docs/index.md:6]: Please restore indentation, as it should be at the same level as the _div_ right above.

File: docs/sass/media-queries.scss [hunk 3]
```diff
@@ -17,21 +18,21 @@ $breakpoints: (
 
 // @include respond-above(s) {}
 @mixin respond-above($breakpoint) {
-    // If the breakpoint exists in the map.
-    @if map-has-key($breakpoints, $breakpoint) {
-        // Get the breakpoint value.
-        $breakpoint-value: map-get($breakpoints, $breakpoint);
+	// If the breakpoint exists in the map.
```
Comment (cyenyxe) [inline docs/sass/media-queries.scss]: Could indentation be made consistent across the different file types? Right now (new) CSS files use tabs, whereas the rest use spaces. I think all our projects favor spaces so let's stick to those.

**Review by cyenyxe — CHANGES_REQUESTED** [review]

File: docs/index.md (hunk 2 above)
Comment (bagnall-fetch) [reply docs/index.md:6]: Fixed to 2 space indents as I think that's what is being requested? My preference is generally 4 spaces.

File: docs/sass/media-queries.scss (hunk 3 above)
Comment (bagnall-fetch) [reply docs/sass/media-queries.scss]: Indent is using spaces.

Comment (bagnall-fetch) [reply docs/sass/media-queries.scss]: CSS files are generated in the build.

**Comment by bagnall-fetch** [comment]
Latest commit has reduced header height in the circumstances described by Josh. To do this, needed to change the side menu so that it does not occupy just the remainder of the viewport height and have its own scroll. Instead, I have adopted a technique similar to new.bbc.co.uk that I showed to Josh, whereby the menu replaces the content in the main flow and the whole screen scrolls. This cannot be done for sites with hero sections however, so header height will not match across all sites.

**Comment by bagnall-fetch** [comment]
I still need to go over the conflicts with Josh.

**Pull request merged**